src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -975,15 +976,17 @@ private bool HasHeader
         /// <param name="hasHeader">Whether the file has a header.</param>
         /// <param name="separatorChar"> The character used as separator between data points in a row. By default the tab character is used as separator.</param>
         /// <param name="dataSample">Allows to expose items that can be used for reading.</param>
-        public TextLoader(IHostEnvironment env, Column[] columns, bool hasHeader = false, char separatorChar = '\t', IMultiStreamSource dataSample = null)
-            : this(env, MakeArgs(columns, hasHeader, new[] { separatorChar }), dataSample)
+        /// <param name="allowSparse">Whether the file can contain numerical vectors in sparse format.</param>
+        /// <param name="allowQuoting">Whether the file can contain numerical vectors in sparse format.</param>
+        public TextLoader(IHostEnvironment env, Column[] columns, bool hasHeader = false, char separatorChar = '\t', IMultiStreamSource dataSample = null, bool allowSparse = false, bool allowQuoting = false)
```

Comment:
The new parameters should use the default constants.

Reply:
Sure.

---
In reply to: [258307575](https://github.com/dotnet/machinelearning/pull/2630#discussion_r258307575) [](ancestors = 258307575)